src/patternfly/components/Popover/popover.scss

```diff
@@ -2,37 +2,40 @@
-  --#{$popover}--m-custom__title-icon--Color: var(--#{$pf-global}--custom-color--200);
+  --#{$popover}--m-danger__title-icon--Color: var(--pf-t--global--icon--color--status--danger--default);
+  --#{$popover}--m-warning__title-icon--Color: var(--pf-t--global--icon--color--status--warning--default);
+  --#{$popover}--m-success__title-icon--Color: var(--pf-t--global--icon--color--status--success--default);
+  --#{$popover}--m-info__title-icon--Color: var(--pf-t--global--icon--color--status--info--default);
+  --#{$popover}--m-custom__title-icon--Color: var(--pf-t--global--icon--color--status--custom--default);
 
   // Title text colors for states
-  --#{$popover}--m-danger__title-text--Color: var(--#{$pf-global}--danger-color--200);
-  --#{$popover}--m-warning__title-text--Color: var(--#{$pf-global}--warning-color--200);
-  --#{$popover}--m-success__title-text--Color: var(--#{$pf-global}--success-color--200);
-  --#{$popover}--m-info__title-text--Color: var(--#{$pf-global}--info-color--200);
-  --#{$popover}--m-custom__title-text--Color: var(--#{$pf-global}--custom-color--300);
+  --#{$popover}--m-danger__title-text--Color: var(--pf-t--global--text--color--status--danger--default);
+  --#{$popover}--m-warning__title-text--Color: var(--pf-t--global--text--color--status--warning--default);
+  --#{$popover}--m-success__title-text--Color: var(--pf-t--global--text--color--status--success--default);
+  --#{$popover}--m-info__title-text--Color: var(--pf-t--global--text--color--status--info--default);
+  --#{$popover}--m-custom__title-text--Color: var(--pf-t--global--text--color--status--custom--default);
 
   // Content
-  --#{$popover}__content--BackgroundColor: var(--#{$pf-global}--BackgroundColor--100);
-  --#{$popover}__content--PaddingTop: var(--#{$pf-global}--spacer--md);
-  --#{$popover}__content--PaddingRight: var(--#{$pf-global}--spacer--md);
-  --#{$popover}__content--PaddingBottom: var(--#{$pf-global}--spacer--md);
-  --#{$popover}__content--PaddingLeft: var(--#{$pf-global}--spacer--md);
+  --#{$popover}__content--BackgroundColor: var(--pf-t--global--background--color--floating--default);
+  --#{$popover}__content--PaddingTop: var(--pf-t--global--spacer--md);
+  --#{$popover}__content--PaddingRight: var(--pf-t--global--spacer--md);
+  --#{$popover}__content--PaddingBottom: var(--pf-t--global--spacer--md);
+  --#{$popover}__content--PaddingLeft: var(--pf-t--global--spacer--md);
+  --#{$popover}__content--BorderRadius: var(--pf-t--global--border--radius--medium);
 
   // Arrow
-  --#{$popover}__arrow--Width: var(--#{$pf-global}--arrow--width-lg);
-  --#{$popover}__arrow--Height: var(--#{$pf-global}--arrow--width-lg);
-  --#{$popover}__arrow--BoxShadow: var(--#{$pf-global}--BoxShadow--lg);
-  --#{$popover}__arrow--BackgroundColor: var(--#{$pf-global}--BackgroundColor--100);
+  --#{$popover}__arrow--Width: #{pf-size-prem(25px)};
+  --#{$popover}__arrow--Height: #{pf-size-prem(25px)};
```

Comment:
There's no arrow tokens, so I just took the value of `--#{$pf-global}--arrow--width-lg` which is what Tooltip has for similar CSS vars

Reply:
Update to same size as Tooltip